src/test/FuzzerImpl.cpp

```diff
@@ -27,12 +28,20 @@ namespace stellar
 namespace FuzzUtils
 {
 auto constexpr FUZZER_MAX_OPERATIONS = 5;
-auto constexpr INITIAL_LUMEN_AND_ASSET_BALANCE = 100000LL;
+auto constexpr INITIAL_ASSET_DISTRIBUTION = 100LL;
 auto constexpr NUMBER_OF_ASSETS_TO_ISSUE = 4;
+auto constexpr FUZZING_FEE = 1;
+auto constexpr FUZZING_RESERVE = 4;
 
 // must be strictly less than 255
 uint8_t constexpr NUMBER_OF_PREGENERATED_ACCOUNTS = 16;
 
+// As a starting point, twice what we need for setup.
+auto constexpr INITIAL_STROOP_BALANCE =
+    2LL * (2 * FUZZING_RESERVE + (FUZZING_FEE + INITIAL_ASSET_DISTRIBUTION) *
```

Comment:
I like this idea.  I've added a revision which factors out the setup parameters from the execution of the operations that perform the setup, so that the configuration can be chosen in one place, and account balances in particular can be computed with knowledge of all relevant configuration parameters.  I've used something like your formula for that balance computation.

I still don't think that the current parameters will be the "final" (inevitably eternally provisional) ones, but I think that this reorganization should make it easy to tweak individual parameters and have any others that need to adapt do so automatically.  I think it will also be extensible to other enhancements, such as starting to use some of the trust line flags.